hw.rb

```diff
@@ -0,0 +1,203 @@
+
+students = [
+    {
+        first_name: 'John',
+        last_name: 'Doe'
+    },
+    {
+        first_name: 'Jane',
+        last_name: 'Doe',
+    },
+    {
+        first_name: 'Jennifer',
+        last_name: 'Aniston',
+    }
+  ]
+  upper_case_full_names = []
+
+
+  students.length.times do |index|
+    upper_case_full_names.push(students[index][:firstname] + " " + students[index][:lastname])
```

Comment:
Great work; but I think here you meant ```[:first_name]```,  ```:[:last_name]``` not ```[:firstname]```, ```[:lastname]```, also you need here to convert the letters from lowercase to uppercase so the code will be : 
```rb
upper_case_full_names = []
students.length.times do |index|
  upper_case_full_names.push((students[index][:first_name] + " " + students[index][:last_name]).upcase)
end
```